src/versionbits.h

```diff
@@ -6,6 +6,7 @@
 #define BITCOIN_VERSIONBITS_H
 
 #include <chain.h>
+#include <consensus/params.h>
```

Comment:
I'm surprised `deploymentstatus.h` doesn't also need to include consensus/params.h, given it defines (non-template) inline functions that call `Consensus::ValidDeployment()`

Reply:
According to https://api.cirrus-ci.com/v1/task/5220634955874304/logs/ci.log, the `deploymentstatus.h` is not parsed by IWYU now.